### ISSUE TITLE
SBIE2335

### ISSUE THREAD

**Issue by ILYAGRISH** [sandboxie-plus/Sandboxie#1971]
### What happened?

An error occurs when installing the program in sandbox. SBIE2335 - 
|Время|        |Сообщение|                                                                          

23:12:12.575   ID процесса 1172: SBIE2335 Initialization failed for process mscorsvw.exe [11 / 5]   

### Download link

Not relevant

### To Reproduce

An error occurs when installing the program in sandbox

### Expected behavior

Non

### What is your Windows edition and version?

Windows 10 Pro 21H1 64-bit

### In which Windows account you have this problem?

User account with UAC protection completely turned off.

### Please mention any installed security software

No

### What version of Sandboxie are you running?

Sandboxie 1.2.2 64 bit

### In which sandbox type you have this problem?

In a Standard isolation sandbox (yellow sandbox icon).

### Where is the program located?

The program is installed only inside a sandbox (NOT in the real system anyway).

### Can you reproduce this problem on an empty sandbox?

I can confirm it also on an empty sandbox.

**Comment by isaak654** [comment]
> Sandboxie 1.2.2 64 bit

No such version currently exists, do you mean Plus 1.0.22 or 1.1.2?

**Comment by ILYAGRISH** [comment]
1.1.2

**Comment by isaak654** [comment]
Could you clarify whether you tried to install a version of .NET Framework or a third-party application that uses it?

**Comment by ILYAGRISH** [comment]
No, I was installing a third-party application that probably uses it. The application is Affinity.Designer.v1.6.4.104. The error occurs at the very end of the installation.

**Comment by ILYAGRISH** [comment]
![image](https://user-images.githubusercontent.com/59709212/175382486-949dc136-df1d-4fee-943e-2f6b3fe2195d.png)

**Comment by ILYAGRISH** [comment]
https://user-images.githubusercontent.com/59709212/175387327-5f84eb9b-aa39-454d-8315-5c503832bc16.mp4

Part 1

**Comment by ILYAGRISH** [comment]
https://user-images.githubusercontent.com/59709212/175387403-5bf7be6e-0684-4b52-afd8-948236aa7bb9.mp4

Part 2
